packs/spell-effects/spell-effect-fiery-body-9th-level.json

```
@@ -4,7 +4,7 @@
     "name": "Spell Effect: Fiery Body (9th Level)",
     "system": {
         "description": {
-            "value": "<p>Granted by @UUID[Compendium.pf2e.spells-srd.Item.Fiery Body]</p>\n<p>You become living flame, giving you fire immunity, resistance 10 to precision damage, and weakness 5 to cold and to water. Any creature that touches you or damages you with an unarmed attack or non-reach melee weapon takes @Damage[4d6[fire]] damage. Your unarmed attacks deal 2d4 additional fire damage, and your fire spells deal one additional die of fire damage (of the same damage die the spell uses). You can cast @UUID[Compendium.pf2e.spells-srd.Item.Ignition] as an innate spell; the casting is reduced from 2 actions to 1. In fire form, you have a fly Speed of 60 feet and don't need to breathe.</p>"
+            "value": "<p>Granted by @UUID[Compendium.pf2e.spells-srd.Item.Fiery Body]</p>\n<p>You gain fire immunity, resistance 10 to precision damage, weakness 5 to cold and to water, and a fly Speed of 60 feet. Any creature that touches you or damages you with an unarmed attack or non-reach melee weapon takes @Damage[4d6[fire]] damage.</p>\n<p>Your unarmed attacks deal 2d4 additional fire damage, and your fire spells deal one additional die of fire damage (of the same damage die the spell uses)."
```

```suggestion
            "value": "<p>Granted by @UUID[Compendium.pf2e.spells-srd.Item.Fiery Body]</p><p>You gain fire immunity, resistance 10 to precision damage, weakness 5 to cold and to water, and a fly Speed of 60 feet. Any creature that touches you or damages you with an unarmed attack or non-reach melee weapon takes @Damage[4d6[fire]] damage.</p>\n<p>Your unarmed attacks deal 2d4 additional fire damage, and your fire spells deal one additional die of fire damage (of the same damage die the spell uses)."
```

Making a meaningless change to force the github pipeline to run 🙃